### PR TITLE
Implement split outputs

### PR DESCRIPTION
**Pull request by ellesmith88**
Added tests for time slicing and area sunset - checks datasets have been subsetted correctly.
Added None as a supported format to allow `SimpleFileNamer` to name files with extension `.dat`

### CODE REVIEW

File: clisops/__init__.py [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+import logging.config
+
+import dask
 from roocs_utils.config import get_config
 
 import clisops
 
 from .__version__ import __author__, __email__, __version__
 
+logging.config.fileConfig("logging.conf")
```
[the remark this answers is not in the document]
Comment (cehbrecht) [reply clisops/__init__.py]: made that change.

[the remark this answers is not in the document]
Comment (ellesmith88) [reply clisops/__init__.py]: That looks good

**Comment by cehbrecht** [comment]
@ellesmith88 Looks like things are fixed and tests are working again. Ok to merge? Just wait for the last travis result ...

**Comment by ellesmith88** [comment]
@cehbrecht Yes, happy to merge if the travis tests all pass.

**Pull request merged**